package-lock.json

```diff
@@ -0,0 +1,3 @@
+{
```

Comment:
Why is there a package-lock.json file in a Spring project?  This should not be here.

Reply:
Hi Mitch, somehow my file got corrupted.  I removed the package-lock.json file and updated the commit name.